fractions/opentracing/src/main/java/org/wildfly/swarm/opentracing/runtime/OpenTracingInstaller.java

```diff
@@ -0,0 +1,37 @@
+package org.wildfly.swarm.opentracing.runtime;
```

Comment:
I don't believe this class is needed at all?

The ServletContextListener being in `deployment` package and attaching that module to users archive with `@DeploymentModule` should be enough